enhancements/network/enable-flows-collection.md

```diff
@@ -0,0 +1,362 @@
+```
+
+#### Console-based flow status and retrieval
+
+If the cluster uses `OVNKubernetes` CNI type, in the side panel of the Openshift
+Console, an "Observability" or "Monitoring" entry would appear inside the
+"Networking" group.
+
+This would lead to a page where you can enable/disable the Network flows, and
+see their status. When the flows' emission is disabled, it would just show an
+"Enable flows" dropdown list, where you can select one of the following
+entries:
+
+```text
+                 +-----------+-+                            
+ Flows emission: |Disabled   |V|                            
+                 |           +-+                            
+                 |IPFix        |                            
+                 +-------------+                            
+```
+
+Once the emission has been enabled, the GUI would show a table with the
+status for each node, analogue to the CLI use case:
+
+```text
+                       +-----------+-+
+       Flows emission: |IPFix      |V|
+                       +-----------+-+
+
+                                
+    +------------------------------------------------+
+    |  NAME       STATUS      ENDPOINT               |
+    +------------------------------------------------+
+    |  node-a     emitting    http://10.1.2.3:30001  |
+    |  node-b     emitting    http://10.4.5.6:30001  | 
+    |  node-c     emitting    http://10.7.8.9:30001  |
+    +------------------------------------------------+
+```
+
+#### Tuning flows' emission parameters
+
+Some setups might lead to an overcongestion of the cluster resources such as the network or the
+flows' ingestion/processing/storage pipeline (e.g. because the size and frequency of reported
+flows is too high).
+
+To minimize it, the user should be able to tune up the flows' cache mechanism in order to control
+the number of records sent to collector.
+
+According to the [OpenVSwitch configuration options](https://www.openvswitch.org/support/dist-docs/ovs-vswitchd.conf.db.5.html),
+there are multiple configuration options that could be exposed to the user:
+
+* `cache_active_timeout`: optional integer, in range 0 to 4,200
+  The maximum period in seconds for which an IPFIX flow record  is
+  cached  and  aggregated  before  being  sent.  If not specified,
+  defaults to 0. If 0, caching is disabled.
+
+* `cache_max_flows`: optional integer, in range 0 to 4,294,967,295
+  The maximum number of IPFIX flow records that can be cached at a
+  time.  If  not  specified,  defaults to 0. If 0, caching is disabled.
+
+* `sampling`: optional integer, in range 1 to 4,294,967,295
+  The  rate  at  which  packets should be sampled and sent to each
+  target collector. If not specified, defaults to 400, which means
+  one  out of 400 packets, on average, will be sent to each target
+  collector.
+
+Other arguments might be considered, such as `enable-input-sampling`, `enable-output-sampling`,
+or `enable-tunnel-sampling`.
+
```

Comment:
Let's add sampling as one of the parameters that can be configured in the CLI and Web Console.